_includes/disclaimer.html

```diff
@@ -1,5 +1,12 @@
-<div class="disclaimer">
+<header class="site-header">
```

Comment:
It's definitely not worth holding up a merge over. ;) So I'll merge it in. But using a class name seems against the point of having `<header>` in the first place (same with `<nav>` or `<footer>`). I think the appropriate selector here is `body > header`, which IIRC selects an immediate child (not a more distant descendant).